commands/networkscommands/commands.go

```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "network",
-			Usage:       "Used for Cloud Networks network operations",
+			Usage:       "Set up new networks to use by servers.",
 			Subcommands: networkcommands.Get(),
 		},
 		{
 			Name:        "subnet",
-			Usage:       "Used for Cloud Networks subnet operations",
+			Usage:       "Allocate IP address blocks, gateways, DNS servers, and host routes to networks.",
 			Subcommands: subnetcommands.Get(),
 		},
 		{
 			Name:        "port",
-			Usage:       "Used for Cloud Networks port operations",
+			Usage:       "Virtual switch ports on logical network switches.",
```

Comment:
...yeah I didn't either

http://docs.rackspace.com/servers/api/v2/cn-devguide/content/conceptPort.html

I've not used any of these features. Some of them are in early availability only.